tests/test_subpackages.py

```diff
@@ -268,6 +268,12 @@ def test_overlapping_files(testing_config, caplog):
     assert sum(int("Exact overlap" in rec.message) for rec in caplog.records) == 1
 
 
+def test_hash_file_contents(testing_config):
```

Comment:
Maybe we don't need this one here if we already have it in `test_post`, though.

Reply:
This was the issue. I had the same test duplicated at two places and I was trying to fix one when the other was failing.